annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -0,0 +1,73 @@
+# Dekorate Autogenerated Notes
+
+To access the Helm annotations or properties you just need to have the following dependency in your
+class path:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>helm-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+Build the project using:
+
+    mvn clean install
+
+You can find the generated Helm artifacts under: `target/classes/META-INF/dekorate/helm/<chart name>/` that should look like:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+
+**Note**: The `<chart name>` is set from either the property `dekorate.helm.name` or the `@HelmChart` annotation.
+
+How can it be used?
+
+First, make sure you have installed [the Helm command line](https://helm.sh/docs/intro/install/) and have connected into a cluster.
```

Comment:
"and have connected into a cluster." --> "and connected/logged to a kubernetes cluster."

Reply:
Changed